### PR TITLE
Fix Dave's image, add size to mini-guide

### PR DESCRIPTION
**Pull request by konklone**
@vdavez' image on the live site is squashed:

before

This crops the image and sizes it down to 250x250 to fix it:

after

I also noted the standard size in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ If you're a new teammate, add yourself to the website by:
 1. Add yourself to the `team.yml` file in 18F/data-private/ and be sure to read [the Privacy section in the README there](https://github.com/18F/data-private#privacy).
 2. Clone this repo either with the GitHub Desktop App or through the command line
 3. Create a branch
-4. Add a photo to the [`assets/images/team`](https://github.com/18F/18f.gsa.gov/tree/staging/assets/images/team) directory that matches the name you used for `name` in the `team.yml` file
+4. Add a **250x250** photo to the [`assets/images/team`](https://github.com/18F/18f.gsa.gov/tree/staging/assets/images/team) directory that matches the name you used for `name` in the `team.yml` file
```
Comment (juliaelman) [inline README.md]: Good addition! :+1: It might be good to change this to add "pixels" to be more explicit.

**Comment by juliaelman** [comment]
@konklone looks good! Added one suggestion to the docs and then I think it's good to merge.

**Comment by konklone** [comment]
:+1: I added `px` to address your suggestion. And in general, feel free to just make the edit yourself directly before merging -- that's one of the reasons I love in-repo branch PRs.

**Comment by juliaelman** [comment]
@konklone will do for next time and thanks for the suggestion!

**Pull request merged**